document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/mapping/mappers/registrationinformation/ApiToRegistrationInformationMapper.java

```diff
@@ -62,30 +74,50 @@ protected void convertDate(CompanyProfileApi companyProfileApi, @MappingTarget R
         if (companyProfileApi != null) {
 
             if (companyProfileApi.getDateOfCreation() != null) {
+                registrationInformation.setDateOfIncorporation(companyProfileApi.getDateOfCreation().
+                        format(DateTimeFormatter.ofPattern(REPORT_DATE_FORMAT)));
+            }
+        }
+    }
 
-                registrationInformation.setDateOfIncorporation(companyProfileApi.getDateOfCreation().format(DateTimeFormatter.ofPattern("d MMMM uuuu")));
+
+    @AfterMapping
+    protected void setIncorporationDateLabel(CompanyProfileApi companyProfileApi, @MappingTarget RegistrationInformation registrationInformation) {
+
+        if (companyProfileApi != null) {
+            if (companyProfileApi.getDateOfCreation() != null) {
```

Comment:
This could be merged with the above if statement `if (companyProfileApi != null && companyProfileApi.getDateOfCreation() != null) {`